repo/plugin.video.nzbdav/resources/lib/nzb_manifest.py

```
@@ -397,63 +421,72 @@ def _reject_entity_declarations(nzb_bytes):
-    """Return main video-file or provisional archive metadata from an NZB XML."""
+def _parse_nzb_root(nzb_bytes):
+    """Parse NZB bytes into an XML root, or None on parse/safety failure."""
     try:
         if _USING_DEFUSEDXML:
             # forbid_dtd=False keeps the standard NZB ``<!DOCTYPE nzb ...>``
             # working; entity/external-reference defenses stay enabled.
-            root = ET.fromstring(nzb_bytes, forbid_dtd=False)
-        else:
-            _reject_entity_declarations(nzb_bytes)
-            root = ET.fromstring(nzb_bytes)
+            return ET.fromstring(nzb_bytes, forbid_dtd=False)
```

Ruff (0.15.20)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)